packages/es-components/src/components/controls/textbox/Textbox.js

```diff
@@ -54,7 +64,11 @@ const Textbox = React.forwardRef((props, ref) => {
         </ValidationIconWrapper>
       )}
       {hasAppend && (
-        <Append addOnTextColor={addOnTextColor} addOnBgColor={addOnBgColor}>
+        <Append
```

Comment:
are there any accessibility concerns introduced by making these clickable since `span` tags are generally not clickable? maybe they should be buttons that are `aria-hidden` since they probably aren't adding much value to people using AT. thoughts?

Reply:
I hadn't considered that, but I really like that idea! I'll give that try.

Reply:
Made that change, it worked out nicely.